Show battery % in split

956MB opened the pull request:
Adds battery percentage above the time and moves the AM/PM below (digital only)

#### Before

![2](https://github.com/user-attachments/assets/9e06dc3c-210d-474d-8bcd-d7dc81fe36f0)

![4](https://github.com/user-attachments/assets/a46a535c-1a4e-4064-a623-0d04d569f679)

#### After

![1](https://github.com/user-attachments/assets/b7db2d8a-2c20-45cf-98ed-b22018ca764d)

![3](https://github.com/user-attachments/assets/9fc7a52a-9bc0-4665-9bec-aa16af32e1b5)

956MB committed:
Show battery % on digital clock

Adds battery percentage above the time and moves the AM/PM below (digital only)

956MB committed:
Also analog

oops... adds the battery above the date when in analog, why not.

mdaskalov commented:
okay, but I would make it configurable and it's also not quite clear what it is.

956MB committed:
Add toggle to battery %

The right key in split mode toggles the battery percentage on/off giving users the option

956MB commented:
> okay, but I would make it configurable and it's also not quite clear what it is.

You're right, it should at least be able to be toggled (it was late and I was rushing). I put the toggle on the `right` key since in split mode it wasn't being used. You could probably also in the future use the `right` to toggle through styles of where things are placed, like month before the day number, day of the week on the top/bottom, battery percent somewhere else, etc.

I think it is clear enough though. All lockscreens usually have at least three core pieces of info: time, date and battery. When people see a "77%" by their clock and a bit later it's "76%", they know. Which is why my first instinct in the main nightstand clock app in mntm was "I wish I could know my battery % as its ticking down".

mdaskalov commented:
merged. Note that month before the day number and showing AM/PM  depends on the Time / Date Format settings!

mdaskalov commented:
The CONFIG_VERSION could be increased but as we have different structure size it can remain unchanged. It will reset on the initial run.
